doc/source/user_guide/missing_data.rst

```diff
@@ -874,7 +874,7 @@ is already ``False``):
    True & pd.NA
 
 
-``NA`` in a boolean context
+``NA`` in a Boolean context
```

Comment:
"boolean" doesn't refer to a `Boolean` class, so it can be left lower case I think?

Reply:
Alright.